src/nvim/ex_cmds.c

```diff
@@ -4822,7 +4822,7 @@ static void helptags_one(char_u *dir, char_u *ext, char_u *tagfname,
 
   // Find all *.txt files.
   size_t dirlen = STRLEN(dir);
-  STRCPY(NameBuff, dir);
+  STRLCPY(NameBuff, dir, MAXPATHL);
```

Comment:
let's only change the lines in this PR. I think we do want to follow this practice in general, but maybe some other PR.